release/models/policy/openconfig-routing-policy.yang

```diff
@@ -54,7 +54,13 @@ module openconfig-routing-policy {
     corresponding action statement is executed.  If the action
     statement has either accept-route or reject-route actions, policy
     evaluation of the current policy definition stops, and no further
-    policy definitions in the chain are evaluated.
+    policy definitions in the chain are evaluated.  If the action
+    statement has the NEXT_STATEMENT action, policy evaluation proceeds
+    to the next statement. If the action statement has the NEXT-POLICY
```

Comment:
```suggestion
    to the next statement. If the actions statement has the NEXT-POLICY
```

Reply:
I think there is just one action per statement?